src/Sulu/Bundle/SecurityBundle/Command/CreateUserCommand.php

```diff
@@ -109,6 +109,9 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $user->setSalt($this->generateSalt());
         $user->setPassword($this->encodePassword($user, $password, $user->getSalt()));
         $user->setLocale($locale);
+        if ($this->isEmailUnique($emailText)) {
```

Comment:
@marcelmoosbrugger shouldn´t you check that in additionally in the interact function?  (if it is not mandatory you could handle it there too)